charts/opensearch/README.md

```diff
@@ -0,0 +1,156 @@
+# OpenSearch Helm Chart
+
+This Helm chart installs [OpenSearch](https://github.com/opensearch-project/OpenSearch) with configurable TLS, RBAC and much more configuration. This chart caters a number of different use cases and setups.
+
+- [Requirements](#requirements)
+- [Installing the chart](#installing)
+- [Uninstalling the chart](#uninstalling)
+- [Configuration](#configuration)
+
+## Requirements
+
+* Kubernetes >= 1.14
+* Helm >= 2.17.0
+* Recommended that you have at least 8 GiB of memory available for this deployment. You can expect the deployment to fail if, say, you have less than 4 GiB of memory available.
+
+## Installing
+
+To install the chart with the release name `my-release`:
+- Switch to opensearch directly after cloning the repo
+  `❯ cd charts/opensearch`
+- Run `❯ helm package .`
+- Install using Helm 3:
+`❯ helm install my-release opensearch-1.0.0.tgz`
+-  Install using Helm 2
+`❯ helm install --name my-release opensearch-1.0.0.tgz`
+
+
+The command deploys OpenSearch with its associated components (data statefulsets, masters, clients) on the Kubernetes cluster in the default configuration.
+
+## Uninstalling
+To delete/uninstall the chart with the release name `my-release`:
```

Comment:
Recommend adding `helm list` to check whether the deployment exist before run `helm delete` or `helm uninstall`.

Also, helm delete has been renamed to uninstall: https://helm.sh/docs/helm/helm_delete/

Reply:
Sounds good. Will make the changes